src/main/java/com/github/jcustenborder/kafka/connect/snmp/AuthenticationProtocol.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright © 2021 Elisa Oyj
+ * Copyright © 2017 Jeremy Custenborder (jcustenborder@gmail.com)
```

Comment:
This copyright is not inherited to new work in new files. You can remove it from this file.

Reply:
Copyright definitions updated with latest commit.